packages/applications-service-api/api/openapi.yaml

```diff
@@ -1006,4 +1039,97 @@ components:
           example: 1
         currentPage:
           type: number
-          example: 1
\ No newline at end of file
+          example: 1
+    SubmissionCreateRequestBody:
+      properties:
+        file:
+          type: string
+          description: File to be uploaded
+          format: binary
+      allOf:
+        - $ref: '#/components/schemas/SubmissionProperties'
+      required:
+        - file
+        - name
+        - email
+        - interestedParty
+        - deadline
+        - submissionType
+    SubmissionResponseBody:
+      allOf:
+        - $ref: '#/components/schemas/SubmissionProperties'
+      properties:
+        caseReference:
+          type: string
+          example: EN010118
+        dateSubmitted:
+          type: string
+          format: date
+          example: 2022-08-02 08:24:05
+        file:
+          type: object
+          properties:
+            name:
+              type: string
+              description: Modified version of the original file name with submissionId appended
+              example: My-Submission-Document-10809.pdf
+            originalName:
+              type: string
+              example: My Submission Document.pdf
+            size:
+              type: integer
+              description: File size in bytes
+              example: 226022
+            md5:
+              type: string
+              example: b1946ac92492d2347c6235b4d2611184
+    SubmissionProperties:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Joe Bloggs
+        email:
+          type: string
+          example: joe@example.org
+        interestedParty:
+          type: boolean
+          description: Is submission on behalf of an interested party
+          example: true
+        ipReference:
+          type: string
+          description: Interested party reference number
+          example: 3287498209
+        deadline:
+          type: string
+          description: Name/title of Deadline
+          example: Deadline 1A
+        submissionType:
+          type: string
+          example: "Other: Biodiversity Ecology and natural Environment"
+        representation:
```

Comment:
Is the idea that if it's a comment we would send it using this endpoint as text and then the API can convert to PDF? 
That would work but we may need to remove the 'required' indicator from file.

Reply:
Yes good call, I have pushed an update making `representation` and `file` optional but have added a description to try and highlight that one of them is required. There doesn't seem to be a non-hacky way of defining such requirements in OpenAPI